app/api/fraud_analysis.py

```
@@ -0,0 +1,20 @@
+    path = "",
+    response_model=FraudResponse,
+    summary="사기 유형 분석"
+)
+async def fraud_analysis(request: FraudRequest):
+    try:
+        answer = await call_gpt(request)
+        response = FraudResponse.model_validate_json(answer)
+        return response
+        
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"사기분석 실패: {e}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)